release/models/aft/openconfig-aft-common.yang

```diff
@@ -348,40 +495,45 @@ submodule openconfig-aft-common {
       default false;
       description
         "Flag that controls pop action, i.e., the top-most MPLS label
-         should be popped from the packet when switched by the system.
+        should be popped from the packet when switched by the system.
 
         The top-most MPLS label associated with pop action is equal to
-        the label key used in 'mpls' AFT 'label-entry' list.";
+        the label key used in 'mpls' AFT 'label-entry' list.
+
+        This node must be supported in addition to the
+        encap-headers/encap-header tree.  A future release of OpenConfig
+        will deprecate this node in favor of the
+        encap-headers/encap-header subtree.";
     }
 
     leaf-list pushed-mpls-label-stack {
-      type oc-mplst:mpls-label;
+      type oc-mplst:mpls-label-stack;
```

Comment:
Hmm... this seems confusing to me. We have a `typedef` that says it should only be used in `leaf-list`. 

Are we re-using this typedef anywhere else? It seems like we probably just want it in one place (i.e., `pushed-mpls-label-stack`)?

Reply:
I would create a `grouping` that just includes this -- rather than the typedef. If you have a typedef that must be used in a leaf-list, then why not just make sure that it is consistently named, and has a consistent description.

Reply:
Done